package.json

```diff
@@ -68,7 +68,7 @@
     "glob-parent": "6.0.2"
   },
   "engines": {
-    "node": ">=8.11.0",
+    "node": ">=8.11.0 <=16.16.0",

```

Comment:
Have you tried 18? Node 14 is end of life next month and 16 end of life on 9/11 of this year